iOS/DuckDuckGo/MainViewController.swift

```diff
@@ -1385,7 +1385,7 @@ class MainViewController: UIViewController {
 
     private func applyWidthToTrayController() {
         if AppWidthObserver.shared.isLargeWidth {
-            self.suggestionTrayController?.float(under: self.viewCoordinator.omniBar.barView.searchContainer)
+            self.suggestionTrayController?.float(withWidth: self.viewCoordinator.omniBar.barView.searchContainerWidth)
```

Comment:
Note this line is different from the original: https://github.com/duckduckgo/apple-browsers/pull/448/files#diff-d48b9c9074291e38d2fedd4c5584963c115538b712507e26f73b1265563c363dL1384. This is because OriginalOmniBarView uses safeAreaInsets properly now, since it's managed by a ViewController. Adding hardcoded padding is not required now.